#include "exceptions.h"
#include "course.h"
#include "iostream"


void mainMenu() {

}

int main() {
	
	try {
		
	}
	catch (...) {

	}



	std::cout << "Course created"<< endl;
	return 0;
}